msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -39,13 +39,13 @@ To complete this tutorial, you need:
 
 If you have an existing personal tab app, make a copy or a branch of your production project for testing and update your App ID in the app manifest to use a new identifier (distinct from the production App ID, for testing).
 
-If you'd like to use sample code to complete this tutorial, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend) to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, then return to this article to update it for Microsoft 365.
+If you'd like to use sample code to complete this tutorial, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend) to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, then return to this article to update it for Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. One of the options available is the *Todo List with backend on Azure* sample.
```

Comment:
```suggestion
If you'd like to use sample code to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend). After you've created a personal tab, return to this article to extend your personal tab across Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. 
```

Reply:
This is more about using sample code vs your own production code. I've revised this section in the latest commit.